index.html

```diff
@@ -23,5 +30,127 @@ <h1 class="headline">Ismail<br />Courr</h1>
       <button class="headline_btn" type="button">Start collaboration</button>
       <img src="/img/image_geometry_2.svg" alt="seperator" />
     </section>
+
+    <!-- Projects section -->
+
+    <section id="projects_section">
+      <h2 class="projects_headline">Projects</h2>
+      <article class="project">
+        <img class="project_img" src="/img/project1.png" alt="project cover" />
+        <div class="project_body">
+          <h3 class="project_name">Project name goes here</h3>
+          <ul class="project_languages">
+            <li class="project_language">HTML/CSS</li>
+            <li class="project_language">Ruby on Rails</li>
+            <li class="project_language">JavaScript</li>
+          </ul>
+          <button class="project_btn">
+            See this Project <i class="fa fa-arrow-right"></i>
+          </button>
+        </div>
+      </article>
+      <article class="project">
+        <img class="project_img" src="/img/project2.png" alt="project cover" />
+        <div class="project_body">
+          <h3 class="project_name">Project name goes here</h3>
+          <ul class="project_languages">
+            <li class="project_language">HTML/CSS</li>
+            <li class="project_language">Ruby on Rails</li>
+            <li class="project_language">JavaScript</li>
+          </ul>
+          <button class="project_btn">
+            See this Project <i class="fa fa-arrow-right"></i>
+          </button>
+        </div>
+      </article>
+      <article class="project">
+        <img class="project_img" src="/img/project3.png" alt="project cover" />
```

Comment:
## [OPTIONAL]

- [ ] I would suggest you use the correct image path because this one is just only working locally but remotely no.
Good Implementation.  `<img class="project_img" src="img/project3.png" alt="project cover" />`

this is how your online version is 👇 
![image](https://user-images.githubusercontent.com/65068771/148438525-26aa00cd-427b-4c6f-8549-e89e4ca684a6.png)
![image](https://user-images.githubusercontent.com/65068771/148438639-f365aa02-e3bb-4366-9447-55e50f78db4d.png)

Reply:
- [ ] [OPTIONAL] @Ismailco, As mentioned by the previous reviewer, It would be proper if you add the images to your GitHub project so that they may be available both locally and remotely for all other people viewing your work. This would make your work more presentable and professional.